docs/telegram.rst

```diff
@@ -41,6 +41,28 @@ Also chat identifier can be configured for each check in ``kibitzr.yml`` using f
         notify:
           - telegram: 118860645
 
+
+Telegram Split
+--------------
+``telegram-split`` will split the report in to multiple messages before sending. The default
+``split-on`` is the new line char ``\n``. The configuration can be one of:
```

Comment:
I think it's better if there will be a single `telegram` notifier with `split-on` option. If the option is not set, don't do the splitting.